install-cockroachdb.md

```diff
@@ -79,10 +66,33 @@ $ make build
 <p>The first time you run <code>make</code>, it can take awhile to download and install various dependencies.</p></li>
 <li><p><a href="/start-a-local-cluster.html">Start a local cluster</a>. </p></li>
 </ol>
+
+<h2 id="use-docker">Use Docker</h2>
+
+<ol>
+<li><p><a href="https://docs.docker.com/mac/step_one/">Install Docker</a>, as described in Docker&#39;s installation instructions for OS X.   </p></li>
+<li><p>Open <strong>Launchpad</strong> and start the <strong>Docker Quickstart Terminal</strong>. This opens a new shell, creates and starts a default Docker virtual machine (VM), and points the terminal environment to this VM.</p></li>
+<li><p>In the shell, pull the official CockroachDB image from <a href="https://hub.docker.com/r/cockroachdb/cockroach/">Docker Hub</a>:</p>
+<div class="highlight"><pre><code class="language-bash" data-lang="bash"><span class="gp">$ </span>docker pull cockroachdb/cockroach
+</code></pre></div></li>
+<li><p>Start a new Docker container and load the CockroachDB image into it:</p>
+<div class="highlight"><pre><code class="language-bash" data-lang="bash"><span class="gp">$ </span>docker run -t -i cockroachdb/cockroach shell
+</code></pre></div></li>
+<li><p><a href="/start-a-local-cluster.html">Start a local cluster</a>.  </p>
+
+<p>When following the <a href="/start-a-local-cluster.html">start a local cluster</a> instructions, be sure to run the commands from within your Docker container. Also, it&#39;s simplest to start the built-in SQL client from within the same container as the CockroachDB cluster, but to do so, you&#39;ll have to start the cluster in the background by adding <code>&amp;</code> to the end of the command:</p>
```

Comment:
OK, that gets me a little further, but I'm still stuck. Here are my steps:

1. Start up Docker Quickstart Terminal. 
2. Start docker container and load cockroach image into it: `docker run -t -i cockroachdb/cockroach shell`.
3. Run `./cockroach start --dev`. Server starts up fine.
4. Open new shell.
5. Run `eval "$(docker-machine env default)"`.
6. Get container id and command:
```
JESSEs-MacBook-Pro:docs jesseseldess$ docker ps
CONTAINER ID        IMAGE                   COMMAND                  CREATED             STATUS              PORTS               NAMES
77e576858691        cockroachdb/cockroach   "/cockroach/cockroach"   3 minutes ago       Up 3 minutes        26257/tcp           pensive_varahamihira
```
7. Run `docker exec -it 77e576858691 /cockroachdb/cockroach sql --dev`. It looks like this works, since I get:
```
JESSEs-MacBook-Pro:docs jesseseldess$ docker exec cfeb9fbfd594 /cockroach/cockroach sql --dev
# Welcome to the cockroach SQL interface.
# All statements must be terminated by a semicolon.
# To exit: CTRL + D.
```
8. But then whenever I try to run sql, I get an error, for example:
cfeb9fbfd594:26257> JESSEs-MacBook-Pro:docs jesseseldess$ show databases;
-bash: show: command not found
```
Any ideas?